manimpango/cmanimpango.pyx

```diff
@@ -13,14 +13,17 @@ class TextSetting:
         font:str,
         slant,
         weight,
-        line_num=-1
+        line_num=-1,
+        color: str=None,
```

Comment:
I think you should change the corresponding implementation of `Text` in tests (see https://github.com/ManimCommunity/ManimPango/blob/main/tests/_manim.py#L110) to add support for this change and add a simple test.